### ISSUE TITLE
Add support for certificate verification

### ISSUE THREAD

**Issue by fkautz** [minio/minio-py#158]
```py
./home/fkautz/.local/lib/python2.7/site-packages/urllib3/connectionpool.py:768: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
```

**Comment by fkautz** [comment]
For easier access:

https://urllib3.readthedocs.org/en/latest/security.html

**Comment by fkautz** [comment]
Fixed with #159.
